internal/pkg/scorecard/resource_handler.go

```diff
@@ -175,7 +175,7 @@ func getPodFromDeployment(depName, namespace string) (pod *v1.Pod, err error) {
 	// instead of the new one.
 	err = wait.PollImmediate(time.Second*1, time.Second*60, func() (bool, error) {
 		pods := &v1.PodList{}
-		err = runtimeClient.List(context.TODO(), &client.ListOptions{LabelSelector: set.AsSelector()}, pods)
+		err = runtimeClient.List(context.TODO(), pods, client.MatchingLabels(set))
```

Comment:
Hi @corinnekrych,
In the https://github.com/operator-framework/operator-sdk/pull/1388/commits/f49484592eecb160259e2b162357c0982676fb92#diff-b38c855ac3ed0aa95294c855ea839f99R176 it was solved as follows. 
```go
err = runtimeClient.List(context.TODO(), pods, client.MatchingLabels(dep.Spec.Selector.MatchLabels))
```